From: Kea <isc-kea@packages.debian.org>
Date: Tue, 19 Feb 2019 12:39:35 +0000
Subject: kea_admin_fix

Removed the fallback to the build version of kea-admin since it will not exist on users hosts.

Patch-Source: https://sources.debian.org/src/isc-kea/1.5.0-2/debian/patches/0003-kea_admin_fix/
---
 src/bin/admin/kea-admin.in | 2 --
 1 file changed, 2 deletions(-)

diff --git a/src/bin/admin/kea-admin.in b/src/bin/admin/kea-admin.in
index 3d391df..dcb8bff 100644
--- a/src/bin/admin/kea-admin.in
+++ b/src/bin/admin/kea-admin.in
@@ -37,8 +37,6 @@ dump_qry=""
 # use build version if it isn't.
 if [ -e @datarootdir@/@PACKAGE_NAME@/scripts/admin-utils.sh ]; then
     . @datarootdir@/@PACKAGE_NAME@/scripts/admin-utils.sh
-else
-    . @abs_top_srcdir@/src/bin/admin/admin-utils.sh
 fi
 
 # Prints out usage version.
